src/main/java/dinkplugin/SettingsManager.java

```diff
@@ -83,19 +84,21 @@ public class SettingsManager {
     private final ConfigManager configManager;
 
     /**
-     * Check whether a username is not on the configured ignore list.
+     * Check whether a username adheres to the configured RSN filter list.
```

Comment:
```suggestion
     * Check whether a username complies with the configured RSN filter list.
```

I think adhere basically never fits in coding, also comply was already used in the readme